moor/lib/src/runtime/query_builder/expressions/expression.dart

```diff
@@ -119,6 +119,14 @@ abstract class Expression<D> implements FunctionParameter {
   }
 }
 
+Expression<bool> existsQuery(BaseSelectStatement select) {
```

Comment:
IMO these top-level functions should be declared in `exists.dart`.

In terms of documentation, we might have something like:

```suggestion
/// The `EXISTS` operator checks whether the [select] subquery returns any rows.
Expression<bool> existsQuery(BaseSelectStatement select) {
```